build.sbt

```diff
@@ -144,6 +145,7 @@ lazy val root =
     .settings(
       name := "enumeratum-root",
       crossVersion := CrossVersion.binary,
+      crossScalaVersions := Nil,
```

Comment:
Why is this set to nil?

Reply:
I've explained that [above](https://github.com/lloydmeta/enumeratum/pull/276#issuecomment-617205784):

> Due to 2.11 not being supported anymore in cats and circe I also had to explicitly disable cross-building on the root project (this is also suggested in the sbt docs) and on the other aggregate projects. I don't know how that might affect publishing so you probably want to have a look at that when you have time to review this.

Reply:
Thanks for clarifying.

Assuming you're talking about https://www.scala-sbt.org/1.x/docs/Cross-Build.html#Cross+building+a+project

> Note: crossScalaVersions must be set to Nil on the root project to avoid double publishing.

I don't ever publish on the root project so I think this isn't necessary. I've never had issues with it either, and if it isn't strictly necessary, I'd rather not change it..

I'm also a bit puzzled because it seemed like the solution [you found](https://github.com/lloydmeta/enumeratum/pull/276#issuecomment-613914980) was to disable 2.11 for those projects?

Reply:
> I'm also a bit puzzled because it seemed like the solution [you found](https://github.com/lloydmeta/enumeratum/pull/276#issuecomment-613914980) was to disable 2.11 for those projects?

Yes but apparently it wasn't enough. The aggregate projects would pull in 2.11 again on these projects. Seems like their `crossScalaVersions` override the settings of the projects they aggregate. Perhaps that's why it's also recommended in the sbt docs.